corehq/apps/userreports/views.py

```diff
@@ -694,7 +694,7 @@ class ReportPreview(BaseDomainView):
     urlname = 'report_preview'
 
     def post(self, request, domain, data_source):
-        report_data = json.loads(six.moves.urllib.parse.unquote(request.body))
+        report_data = json.loads(six.moves.urllib.parse.unquote(request.body.decode('utf-8')))
```

Comment:
this is surprising to me. from my understanding `request.body` is a byte string? I would have expected that to come from django and would have tough django made that a unicode object (especially in py3) do you know why that's not the case?

Reply:
Django says it's a byte string: https://docs.djangoproject.com/en/2.1/ref/request-response/#django.http.HttpRequest.body

Reply:
cool. their explanation makes sense. It's possible that we could use POST to get around that, but there's no way I'd be comfortable making that change in reports without more testing. thanks for the docs